lib/github-pages.rb

```diff
@@ -7,29 +7,30 @@ class GitHubPages
   def self.gems
     {
       # Jekyll
-      "jekyll"                => "2.4.0",
+      "jekyll"                => "3.0.0",
       "jekyll-coffeescript"   => "1.0.1",
       "jekyll-sass-converter" => "1.3.0",
+      "jekyll-paginate"       => "1.1.0",
```

Comment:
Both jekyll-paginate and jekyll-coffeescript can be moved down to "Plugins" as they're no longer runtime dependencies of Jekyll. :)